chore: content update adding margin to component properties

twjeffery opened the pull request.

twjeffery commented:
This is for this issue: [#1862](https://github.com/GovAlta/ui-components/issues/1862
)

ArakTaiRoth commented:
@twjeffery This is missing the following components:

- [ ] Block
- [ ] Button Group
- [ ] Form Stepper
- [ ] Grid
- [ ] Icon
- [ ] Icon Button
- [ ] Pagination
- [ ] Popover
- [ ] Skeleton
- [ ] Table
- [ ] Tooltip

twjeffery commented:
> @twjeffery This is missing the following components:
> 
> * [ ]  Block
> * [ ]  Button Group
> * [ ]  Form Stepper
> * [ ]  Grid
> * [ ]  Icon
> * [ ]  Icon Button
> * [ ]  Pagination
> * [ ]  Popover
> * [ ]  Skeleton
> * [ ]  Table
> * [ ]  Tooltip

Do those all accept margin properties? If so I can update the docs

twjeffery commented:
@ArakTaiRoth I added the missing component docs.

twjeffery commented:
@ArakTaiRoth I added the missing component docs, should all be there now. I also updated what I had done. I realized that I didn't need to show two versions for each React and Angular, I removed that everywhere to simplify.

twjeffery commented on src/routes/components/Radio.tsx:
```diff
@@ -184,6 +184,11 @@ export default function RadioPage() {
       type: "string",
       description: "Defines how the radio item will be translated for the screen reader.",
     },
+    {
```
Added to radio group

twjeffery commented on src/routes/components/FileUploader.tsx:
```diff
@@ -129,6 +129,11 @@ export default function FileUploaderPage() {
       type: "(file: File) => void",
       description: "Event fired for each file selected",
     },
+    {
```
removed from file uploader

twjeffery commented:
@vanessatran-ddi I addressed all of the comments, should be good now
